src/main/java/com/linkedin/kmf/services/ConsumerFactoryImpl.java

```diff
@@ -34,9 +37,16 @@
       new String[] {ConsumeServiceConfig.BOOTSTRAP_SERVERS_CONFIG, ConsumeServiceConfig.ZOOKEEPER_CONNECT_CONFIG};
   private int _latencySlaMs;
   private static AdminClient adminClient;
```

Comment:
1. Can we name this field `_ adminClient` instead of `adminClient` so that naming conventions are consistent across all fields?
2. Can we add `final` to this field unless there is some reason not to do so?

Reply:
1. We should not do that. class-level static, non-final variables follow a naming convention where the underscore doesn't prefix the variable name. checkstyle is that is should match pattern '^[a-z][a-zA-Z0-9]*$'. [StaticVariableName] 
2. We also should not do that. That would force KMF to initialize the AdminClient in the class level declaration and not in the constructor. Let's create AdminClient object in the constructor.

Reply:
Please refer to https://iwww.corp.linkedin.com/wiki/cf/display/ENGS/Java+Coding+Conventions

Reply:
I see. I did not notice that the  AdminClient is a static field

Reply:
No problem